### PR TITLE
feat: token list management

### PR DESCRIPTION
**Pull request by Lanttcat**
- feat: add manage token list bar component
- feat: add mode to control token list status

## Description

Closes #MF-485 #MF-1523

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

**Comment by guanbinrui** [comment]
https://user-images.githubusercontent.com/52657989/177150434-0dae03a7-77a5-4ecd-8cd5-da75d56b850e.mov

The icons are blinking when the popup opened.

File: packages/theme/src/Components/SearchableList/SearchableList.tsx [hunk 1]
```diff
@@ -105,6 +108,14 @@ export function SearchableList<T extends {}>({
                                     <Icons.Search />
                                 </InputAdornment>
                             ),
+                            endAdornment: keyword ? (
+                                <InputAdornment
+                                    position="end"
+                                    className={classes.closeIcon}
+                                    onClick={() => handleSearch('')}>
+                                    <Icons.FilledClose size={18} />
```
Comment (UncleBill) [inline packages/theme/src/Components/SearchableList/SearchableList.tsx]: Use a proper name.

```suggestion
                                    className={classes.clearIcon}
                                    onClick={() => handleSearch('')}>
                                    <Icons.Clear size={18} />
```

File: packages/plugin-infra/src/web3-state/Token.ts [hunk 2]
```diff
@@ -11,8 +11,16 @@ import {
 import type { Plugin } from '../types'
 
 export interface TokenStorage<ChainId, SchemaType> {
+    /**
+     * @deprecated Use fungibleTokenList instead.
+     */
     fungibleTokens: Array<FungibleToken<ChainId, SchemaType>>
+    /**
+     * @deprecated Use nonFungibleTokenList instead.
+     */
```
Comment (guanbinrui) [inline packages/plugin-infra/src/web3-state/Token.ts]: Just remove them. No need to consider the backward-compatible problem here.

File: packages/shared/src/locales/en-US.json [hunk 3]
```diff
@@ -3,10 +3,13 @@
     "import": "Import",
     "balance": "Balance",
     "select_token": "Select a Token",
-    "erc20_token_list_placeholder": "Token name or contract address",
+    "erc20_token_list_placeholder": "Name or Contract address eg.USDC or 0x234...",
```
Comment (guanbinrui) [inline packages/shared/src/locales/en-US.json]:
```suggestion
    "erc20_token_list_placeholder": "Name or Contract address e.g., USDC or 0x234...",
```

**Comment by guanbinrui** [comment]
image

It needs to highlight token icons in dark mode.

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**